### PR TITLE
Bunq/tinker_php#23-update sdk version to major 1

### PR DESCRIPTION
**Pull request by OGKevin**
References #23

### CODE REVIEW

**Comment by OGKevin** [comment]
@JordyHeemskerk

File: src/BunqLib.php [hunk 1]
```diff
@@ -402,8 +403,8 @@ public function linkCardToBankAccount(Card $card, MonetaryAccountBank $monetaryA
             null, /* status */
             null, /* limit */
             null, /* magStripePermission */
-            null, /* countryPermission */
-            $monetaryAccount->getId()
+            null,
```
Comment (JordyHeemskerk) [inline src/BunqLib.php]: Missing comment

File: src/BunqLib.php [hunk 2]
```diff
@@ -402,8 +403,8 @@ public function linkCardToBankAccount(Card $card, MonetaryAccountBank $monetaryA
...
+            [new CardPinAssignment('PRIMARY', null, $monetaryAccount->getId())]
```
Comment (JordyHeemskerk) [inline src/BunqLib.php]: Also missing comment on what argument is used for.

**Review by JordyHeemskerk — CHANGES_REQUESTED** [review]

File: src/BunqLib.php (hunk 2 above)
Comment (OGKevin) [reply src/BunqLib.php]: what argument i dont follow ? No other arguments in this file has this.

File: src/BunqLib.php [hunk 3]
```diff
@@ -403,7 +404,13 @@ public function linkCardToBankAccount(Card $card, MonetaryAccountBank $monetaryA
             null, /* limit */
             null, /* magStripePermission */
             null, /* countryPermission */
-            $monetaryAccount->getId()
+            [
+                new CardPinAssignment(
+                    'PRIMARY',
```
Comment (JordyHeemskerk) [inline src/BunqLib.php]: extract const

**Review by JordyHeemskerk — DISMISSED** [review]

**Review by JordyHeemskerk — APPROVED** [review]

**Comment by OGKevin** [comment]
@andrederoos

**Pull request merged**